### ISSUE TITLE
cannot reproduce the result of open-sourced model

### ISSUE THREAD

**Issue by zzwjames** [SalesforceAIResearch/Elastic-Reasoning#2]
hi, thanks for your open-sourced code and model. I found that I could not reproduce the result tested on your open-sourced model. But i followed your code and got results similar to that reported in the paper. I am just wondering does your open-sourced model trained on the same setting as shown in the code? Thanks!

Best,

**Comment by yuhuixu1993** [comment]
@zzwjames, thanks for the interest of our paper. Yes, the open-sourced models are trained on the same setting as shown in the code. Can you share the model name that you can not reproduce and your result?

**Comment by zzwjames** [comment]
> [@zzwjames](https://github.com/zzwjames), thanks for the interest of our paper. Yes, the open-sourced models are trained on the same setting as shown in the code. Can you share the model name that you can not reproduce and your result?

Hi, Thanks for your reply. I tried to reproduce the results on math reasoning. The released model could achieve 17.5% for pass@1 and 46.7 for pass@16 on AIME, but my trained model get 14% for pass@1 and 40.0 for pass@16. I understand this could be due to randomness and I am just wondering did you pick the model from last epoch or pick based on the validation accuracy? Thanks!

**Comment by yuhuixu1993** [comment]
@zzwjames ,hi, Yes, we pick the checkpoints based on the validation accuracy.
